src/coreclr/jit/lsra.cpp

```diff
@@ -4948,7 +4949,7 @@ void LinearScan::freeRegisters(regMaskTP regsToFree)
         if (regRecord->assignedInterval != nullptr && (regRecord->assignedInterval->registerType == TYP_DOUBLE))
         {
             assert(genIsValidDoubleReg(nextReg));
-            regsToFree ^= genRegMask(regNumber(nextReg + 1));
+            regsToFree ^= regNumber(nextReg + 1);
```

Comment:
```suggestion
            regsToFree.RemoveRegNumFromMask(regNumber(nextReg + 1));
```

Reply:
(Note that both arm64/x64 have efficient instructions to clear a bit, so I don't think the xor is actually saving anything)

Reply:
yeah, i was just trying to mimic the existing operations without thinking about efficient way to clear a bit. I have replaced this with `RemoveRegNumFromMask`.